src/includes/functions/math/complex/src/pow.js

```diff
@@ -66,11 +63,21 @@ pow(complex) {
 		return new BigNumber(Math.log(a / Math.cos(Math.atan2(b, a))))
 	}
 
+	let c;
+	let d;
+
+	if (complex.isComplex) {
+    	c = complex.a;
+    	d = complex.b;
+	} else if (typeof complex === 'number') {
+		c = complex;
+		d = 0;
+	} else {
+		throw "[TheoremJS]: Unsupported typeof power"
+	}
```

Comment:
If argument is a complex number, run as before, if `typeof` the argument is a number, let `real = n` and `im = 0`, else throw type error.